pandas/core/arrays/categorical.py

```diff
@@ -416,12 +416,12 @@ def categories(self):
 
         See Also
         --------
-        rename_categories
-        reorder_categories
-        add_categories
-        remove_categories
-        remove_unused_categories
-        set_categories
+        rename_categories : Rename categories.
+        reorder_categories : Reorder categories.
+        add_categories : Add new categories.
+        remove_categories : Remove the specified categories.
+        remove_unused_categories : Remove categories which are not used.
+        set_categories : Set the categories to the specified categories.
```

Comment:
Not sure if this reads a bit better not repeating the word categories?
```suggestion
        set_categories : Set the categories to the specified ones.
```